hyperstart_agent.go

```diff
@@ -53,6 +53,16 @@ func (c *HyperConfig) Logger() *logrus.Entry {
 	return virtLog.WithField("subsystem", "hyperstart")
 }
 
+//Generic function for creating a named-id for passing on the hypervisor commandline
```

Comment:
That's not hyperstart specific afaict. Please move it to e.g. hypervisor.go.

Reply:
@egernst I think you put that in the wrong file. We had agreed the most simple/generic would be `hypervisor.go`.